torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -282,3 +282,32 @@ def convert_color_space(
         return inpt.to_color_space(color_space, copy=copy)
     else:
         return convert_color_space_image_pil(inpt, color_space, copy=copy)
+
+
+def convert_dtype_image_tensor(
+    image: torch.Tensor, dtype: torch.dtype = torch.float, copy: bool = True
```

Comment:
I've opted to add a `copy` flag here in accordance with the other conversion transforms.

Reply:
I understand your intention to support `copy` is to align with other APIs but this introduces unnecessary clone calls in our existing Transforms. Under which conditions do we even want to this? If we don't have a real use-case for this, I would prefer to remove it.